cache/README.md

```diff
@@ -0,0 +1,33 @@
+# FightPandemics ElastiCache Infrastructure
+
+This repo contains the Terraform code for setting up the FightPandemics ElastiCache infrastructure.
+
+Note that the Terraform state for this module is separate from the main infrastructure Terraform state.
+
+## Prerequisites
+
+1. Install Terraform v0.12.29 by downloading the package for your operating system [here](https://www.terraform.io/downloads.html). Alternatively, you can run Terraform in [Docker](https://hub.docker.com/r/hashicorp/terraform). NOTE: Please install Terraform v0.12.29 as v0.13 has some breaking changes.
```

Comment:
[here] Terraform download page has links to v.013.  For v.0.12.29 should we suggest chocolaty/homebrew for a specific version?

Reply:
We could suggest those two package managers, although I was able to download the binary for 0.12.29 [here](https://releases.hashicorp.com/terraform/). Perhaps we should just link to that version directly. I don't know if using a package manager is a good idea as there is less control over managing multiple versions of Terraform. At work I need to use 0.13.x for some of the projects, so I have both 0.12.29 and 0.13.x installed on my machine. But I have renamed them to `terraform0.12.29` and `terraform0.13` to differentiate between the two.